scripts/benchmarks-ci.sh

```diff
@@ -44,6 +46,11 @@ fi
 
 for pallet in ${pallets[@]}
 do
+	# a little hack for xcm benchmarks
+	output_file="${pallet//::/_}"
```

Comment:
I think we can pull this out into a new MR.

Reply:
I am afraid we cannot, this is going to break current benchmark generation, this PR should contain the change in order to make sure we don't end up in a pickle where `benchmarks-ci.sh` generates broken filenames.

Reply:
Note that this has already been done for the benchmarks bot a while ago.